e2e-tests/cypress/e2e/journeys/insurance/your-business/check-your-answers/check-your-answers-summary-list/check-your-answers-summary-list.spec.js

```diff
@@ -61,6 +61,7 @@ context('Insurance - Your business - Check your answers - Summary list - your bu
       task.link().click();
 
       cy.completeAndSubmitCompanyDetails();
+      cy.completeAndSubmitYourContact();
```

Comment:
Lots of repitition for these commands, could we use `cy.completeAndSubmitNatureOfYourBusiness`?

Reply:
can this be done later? there will be some changes once the page is autopopulated